docs/CLI-Reference.md

```diff
@@ -449,7 +449,7 @@ Usage: `cl <command> <arguments>`
 
 ## Commands for managing server:
   ### workers:
-    Display worker information of this CodaLab instance. Root user only.
+    Display worker information of this CodaLab instance. If not root user, only display information about your connected workers.
```

Comment:
Use the description from above, but add: "For the root user, display all workers."

Reply:
Note that the pre-commit script will overwrite this change every time to match the output of `cl -h -m` (i.e., removing `For the root user, display all workers.`).